packages/terra-form-input/src/terra-dev-site/doc/form-input/UpgradeGuide.3.doc.mdx

```diff
@@ -1,3 +1,5 @@
+import { Badge } from 'terra-form-input/package.json?dev-site-package';
```

Comment:
The badge is imported but never used

Reply:
Updated here https://github.com/cerner/terra-core/pull/2830/commits/631d0f64c7f790d95ee1f881a888aab064203d47.